global.json

```diff
@@ -1,13 +1,13 @@
 {
     "sdk": 
     {
-        "version": "6.0.403",
-        "rollForward": "patch"
+        "version": "7.0.302",
+        "rollForward": "major"
```

Comment:
I don't think we want `major`.  This will allow an `8.x` to be used, which cannot build `net6.0-android` packages.